lib/llvm-opt-transformer.ts

```diff
@@ -0,0 +1,82 @@
+import {Transform, TransformCallback} from 'stream';
```

Comment:
What license is this file even under?

Reply:
https://github.com/jaredwy/compiler-opt-info/blob/master/LICENSE MIT, should probably be added to the header

Reply:
We used to have a check for this - did we lose it along the way?

Reply:
Oh I bet we did not reenable it after the prettier changes

Reply:
#3619 renables the license checks